examples/Phonon_Caustics/include/Caustic_PhononConfigManager.hh

```diff
@@ -0,0 +1,47 @@
+/***********************************************************************\
+ * This software is licensed under the terms of the GNU General Public *
+ * License version 3 or later. See G4CMP/LICENSE for the full license. *
+\***********************************************************************/
+
+#ifndef Caustic_PhononConfigManager_hh
+#define Caustic_PhononConfigManager_hh 1
+//20240110 Israel Hernandez -- Illinois Institute of Technology, Quantum Science Center and Fermilab
+
+#include "globals.hh"
```

Comment:
Geant4 recommends avoiding the use of "globals.hh" in .hh files.  To get access to the basic data types, use
```
    #include "G4Types.hh"
    #include "G4String.hh"
```

Consider this comment as applying to all of the .hh files.